server/storage/wal/version.go

```diff
@@ -113,11 +113,12 @@ func visitEntryData(entryType raftpb.EntryType, data []byte, visitor Visitor) er
 		}
 		msg = proto.MessageReflect(&raftReq)
 		if raftReq.ClusterVersionSet != nil {
```

Comment:
I think you can remove the whole block for `ClusterVersionSet` and just move the call `return visitMessage(msg, visitor)` from line 145 here.

Reply:
Updated. 

Note we can't move `return visitMessage(msg, visitor)` from line 145 here, otherwise we will have to add a meaningless "`return nil`" at the end of the function.

Reply:
OK